sys/include/net/gnrc/netif.h

```diff
@@ -75,6 +88,21 @@ typedef struct {
      * @see net_gnrc_netif_flags
      */
     uint32_t flags;
+#if defined(MODULE_GNRC_NETIF_ASYNC_EVENTS) || DOXYGEN
+    /**
+     * @brief   Event queue for asynchronous events
+     */
+    event_queue_t evq;
+    /**
+     * @brief   Pointer to event instance table
+     *
+     * This pointer is a way to pass an event instance allocated on the stack by
+     * the netif thread to the IRQ event handler called by the netdev driver
+     * while still keeping the allocation of specific events an implementation
+     * detail of gnrc_netif.c
```

Comment:
I've read this sentence now three times and still find it hard to grasp it. Can you split this up into smaller sentences maybe? Also I think, if I understand it correctly, that this is more an implementation note, and should thus not be in the detailed section, but in the `@note` section.